tests/ludwig/encoders/test_text_encoders.py

```diff
@@ -158,6 +158,51 @@ def test_hf_ludwig_model_reduce_options(tmpdir, csv_filename, encoder_name, redu
         model.train(dataset=rel_path, output_directory=tmpdir)
 
 
+@pytest.mark.parametrize(
+    "pretrained_model_name_or_path",
+    [
+        "hf-internal-testing/tiny-random-bloom",
+        "hf-internal-testing/tiny-random-OPTModel",
+        "hf-internal-testing/tiny-random-GPTJModel",
+    ],
+)
+def test_hf_ludwig_model_auto_transformers(tmpdir, csv_filename, pretrained_model_name_or_path):
```

Comment:
Could you mark this with `@pytest.mark.slow`?

Reply:
I think this should only take 30s, so I would want to run it on every commit if we can get away with it.